contracts/StablecoinConverter.sol

```diff
@@ -231,6 +234,21 @@ contract StablecoinConverter is EpochTokenLocker {
         }
     }
 
+    function evaluateUtility(uint128 execBuy, Order memory order) internal view returns(uint128) {
+        // Utility = ((execBuyAmt * order.sellAmt - scaledSellAmount * order.buyAmt) * price.buyToken) / order.sellAmt
+        uint256 scaledSellAmount = getExecutedSellAmount(
```

Comment:
I'm not a huge fan of the term "scaled sell amount". It reminds me of taking the sellAmount and scaling it up or down. What we really scale here is how much fee is deducted.

Maybe `getExecutedSellAmount` should take `fee Denominator` instead of `scale` as an argument. In the regular case we use the original denominator, and for utility calculation we scale the fee beforehand.

Reply:
Yes, I could do this!